ui/src/app/machines/views/MachineDetails/MachineConfiguration/PowerForm/PowerFormFields/PowerFormFields.tsx

```diff
@@ -1,67 +1,31 @@
-import { Col, Notification, Row } from "@canonical/react-components";
-import { useFormikContext } from "formik";
-import { useSelector } from "react-redux";
+import { Col, Row } from "@canonical/react-components";
 
 import type { PowerFormValues } from "../PowerForm";
 
 import PowerTypeFields from "app/base/components/PowerTypeFields";
-import { useIsRackControllerConnected } from "app/base/hooks";
 import { PowerTypeNames } from "app/store/general/constants";
-import { powerTypes as powerTypesSelectors } from "app/store/general/selectors";
-import { getPowerTypeFromName } from "app/store/general/utils";
 import type { MachineDetails } from "app/store/machine/types";
 import { getMachineFieldScopes } from "app/store/machine/utils";
 
 type Props = {
-  editing: boolean;
   machine: MachineDetails;
 };
 
-const PowerFormFields = ({ editing, machine }: Props): JSX.Element => {
-  const powerTypes = useSelector(powerTypesSelectors.get);
-  const { values } = useFormikContext<PowerFormValues>();
-  const isRackControllerConnected = useIsRackControllerConnected();
-  const powerType = getPowerTypeFromName(powerTypes, values.powerType);
-  const machineInPod = Boolean(machine.pod);
+const PowerFormFields = ({ machine }: Props): JSX.Element => {
+  const machineInPod = !!machine.pod;
```

Comment:
I wonder what made you decide to change the syntax from `Boolean` to double negation `!!`?

Also, do you think it'd be worth renaming this to include an `is` prefix, e.g. `isMachineInPod`.
That'd be consistent with `canEditCertificate`, `initialShouldGenerateCert` which both include a word indicating a boolean value.

Reply:
Good point! Updating now.

I actually prefer using `Boolean()` since it's clearer, but looks like it was a casualty in this rewrite where I reverted to using `!!` as shorthand, oops.